### ISSUE TITLE
Deprecate and remove view_types from the course definition

### ISSUE THREAD

**Issue by jaguarfi** [apluslms/mooc-grader#60]
Defining relative imports like `view_type: .exercises.custom` should not be allowed as it's A potential security issue (running unaudited code).

Basically, grader shouldn't run any code from course repository without sandbox.

**Comment by Mankro** [comment]
Blocked by: some courses depend on custom view types and there is no solution running view type code in a sandbox. 

This issue has low priority since we trust that no teacher tries to abuse the system with custom view types.

**Comment by raphendyr** [comment]
> This issue has low priority since we trust that no teacher tries to abuse the system with custom view types.

This is true, however it also introduces blocker for wide spread use. For now, I would presume this will get fixed with a next generation of the grader.

To help with the issue, one can create a dedicated instance of the MOOC-Grader for a trust group, thus limiting the potential problems.
